src/Microsoft.Health.Fhir.Stu3.Core/Features/Validation/ModelAttributeValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Hl7.Fhir.Validation;
+using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Core.Features.Validation
+{
+    public class ModelAttributeValidator : IModelAttributeValidator
```

Comment:
Both R4 and STU3 version are the same. Could it just be implemented once in the shared project?